compiler/transforms/accfg_config_overlap.py

```diff
@@ -9,12 +9,17 @@
     op_type_rewrite_pattern,
 )
 from xdsl.rewriter import InsertPoint
-from xdsl.traits import is_side_effect_free
+from xdsl.traits import NoMemoryEffect
 
 from compiler.dialects import accfg
 from compiler.inference.helpers import iter_ops_range, previous_ops_of
 from compiler.inference.scoped_setups import get_scoped_setup_inputs
 
+# temporary fix until we update xDSL again
+from xdsl.dialects import memref
+
+setattr(memref.ExtractStridedMetaDataOp, "traits", frozenset([NoMemoryEffect()]))
+
```

Comment:
This was fixed in https://github.com/xdslproject/xdsl/commit/e25b55a1c56ad64316d1301708b7c015913f1853

We can drop it